module/os/freebsd/zfs/zfs_vnops.c

```diff
@@ -5941,7 +5941,7 @@ zfs_getextattr(struct vop_getextattr_args *ap)
 	flags = FREAD;
 	NDINIT_ATVP(&nd, LOOKUP, NOFOLLOW, UIO_SYSSPACE, attrname,
 	    xvp, td);
-	error = vn_open_cred(&nd, &flags, VN_OPEN_INVFS, 0, ap->a_cred, NULL);
+	error = vn_open_cred(&nd, &flags, 0, VN_OPEN_INVFS, ap->a_cred, NULL);
```

Comment:
It looks like we should also be passing `0600` here rather than omitting the mode.

https://github.com/openzfs/zfs/blob/b29e31d80d6cb78dbd889e9b529333944b4c3ba1/module/os/freebsd/zfs/zfs_vnops.c#L6094